napari/_qt/range_slider/tests/range_slider_tests.py

```diff
@@ -0,0 +1,37 @@
+"""
+ Interactive test for range slider.
+"""
+
+import sys
+
+from PyQt5.QtWidgets import QApplication
+
+# starts the QT event loop
+from napari._qt.range_slider.range_slider import QVRangeSlider
```

Comment:
should be relative import not absolute import

Reply:
seems like this is more complex than i thought, and making a relative import throws the following error:

```python
python napari/_qt/range_slider/tests/range_slider_tests.py
Traceback (most recent call last):
  File "napari/_qt/range_slider/tests/range_slider_tests.py", line 6, in <module>
    from ..range_slider.range_slider import QVRangeSlider
ValueError: attempted relative import beyond top-level package
```

The following stack overflow post might shed some light on this
https://stackoverflow.com/questions/6323860/sibling-package-imports/50193944#50193944

Any thoughts @kne42?

Reply:
That's because you are using `python` instead of `pytest`. You could also try `python -m` but don't quote me on that.

Reply:
Hmm, we should probably switch over to `pytest` like everything else then, right?